apple/internal/processor.bzl

```diff
@@ -317,6 +339,11 @@ def _bundle_partial_outputs_files(
                     # Skip files for locales that aren't in the locales for the base resources.
                     continue
 
+            if locales_to_exclude:
+                locale = bundle_paths.locale_for_path(parent_dir)
+                if locale and sets.contains(locales_to_exclude, locale):
+                    continue
```

Comment:
Can this be put closer to the portion that does the `apple.locales_to_include` handling or is it not possible?

https://github.com/bazelbuild/rules_apple/blob/2c5e786fbcef8430d48c66c04b249937e4c8bcf2/apple/internal/partials/resources.bzl#L233-L235

And:

https://github.com/bazelbuild/rules_apple/blob/2c5e786fbcef8430d48c66c04b249937e4c8bcf2/apple/internal/partials/resources.bzl#L240-L246

Reply:
oh yeah, that definitely will be a better place! thanks, updated!